packages/esm-patient-registration-app/src/patient-registration/section/additional-information/additional-information.resource.tsx

```diff
@@ -0,0 +1,20 @@
+import { FetchResponse, openmrsFetch } from '@openmrs/esm-framework';
+import useSWR from 'swr';
+import { ConceptAnswers, ConceptResponse, PersonAttributeTypeResponse } from '../../patient-registration-types';
+
+export function usePersonAttributeType(personAttributeTypeUuid: string): [PersonAttributeTypeResponse, boolean] {
+  const { data, error } = useSWR<FetchResponse<PersonAttributeTypeResponse>>(
+    `/ws/rest/v1/personattributetype/${personAttributeTypeUuid}`,
+    openmrsFetch,
+  );
+  return [data?.data, !data && !error];
```

Comment:
unclear what the second return value represents. No data and no error, is this supposed to be the 'loading' state? Might help to have some docstring clarify